charts/kubescape-operator/README.md

```diff
@@ -1212,3 +1213,13 @@ To retrieve the `NetworkNeighbors` CRD for a workload, you can run the following
 ```bash
 kubectl -n <namespace> get networkneighbors <workload-kind>-<workload-name> -o yaml
 ```
+---
+## Common Issues
+* Error starting the container watcher - `(fanotify)`.
+
+  This error is usually caused by the `node-agent` not being able to find `runc` in any of the default paths.
+  This can be fixed by adding the path of `runc` to the global configuration [here](#values).
+  If you aren't sure where `runc` is located, you can run the following command to find it:
```

Comment:
where should I run this command?
- ssh to node: usually not available
- node-agent pod: impossible since we're distroless
- temporary privileged pod: yes, but you should give the correct kubectl invocation